tools/c7n_azure/requirements.txt

```diff
@@ -1,4 +1,5 @@
 vcrpy==1.11.0
 vcrpy_unittest
 adal~=0.5.0
+mock==2.0.0
```

Comment:
I think the general project approach is to not pin to a version unless we have a known issue that requires it.  Maybe we should just remove the `==2.0.0` here.